services/lumberjack/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "lumberjack",
+  "version": "0.1.0",
+  "main": "lib/service.js",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "start": "node ./bin/start-service",
+    "build": "babel src --out-dir lib",
+    "build-msg": "mkdir -p lib && pbjs -t static-module --es6 --keep-case -o src/messages.js src/messages/*.proto",
+    "test": "jest",
+    "lint": "eslint src test --ignore-pattern src/messages.js"
+  },
+  "dependencies": {
+    "common-nodejs": "file:src/common",
+    "dockerode": "^2.5.7",
+    "influx": "^5.0.7",
+    "kleur": "^3.0.3",
+    "koa": "^2.6.2",
+    "koa-protobuf": "^0.1.0",
+    "koa-router": "^7.4.0",
```

Comment:
Should remove unused dependencies, `kluer`, `koa`, `koa-protobuf`, and `koa-router`.